UnityProject/Assets/Scripts/UI/Core/OptionsMenu/ThemeOptions/ThemeManager.cs

```diff
@@ -74,6 +77,11 @@ void IInitialise.Initialise()
 		        PlayerPrefs.SetInt(PlayerPrefKeys.MentionSoundIndex, 0);
 	        }
 
+	        if (PlayerPrefs.HasKey(PlayerPrefKeys.EnableClassicHoverTooltip) == false)
+	        {
+		        ClassicHoverTooltipToggle(false);
```

Comment:
couldn't this just be
```cs
ClassicHoverTooltipToggle(PlayerPrefs.HasKey(PlayerPrefKeys.EnableClassicHoverTooltip))
```
No need for the if

Reply:
I'm enabling this by default for now for next playtest